### PR TITLE
20 - Basic Github actions

### PR DESCRIPTION
**Pull request by dinakar29**
Track Issue #20

Starter Github actions to enhance DX (developer experience).
Most of these have been borrowed from https://github.com/r-lib/actions/tree/master/examples and some have been lifted-and-shifted from CircleCI

### CODE REVIEW

**Comment by dinakar29** [comment]
The commands workflow will only run after it's been merged into the `master` branch, per this comment: https://github.community/t/on-issue-comment-events-are-not-triggering-workflows/16784/4

File: .github/workflows/pr-commands.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          git config --local user.name "GitHub Actions"
+          git add man/\* NAMESPACE
+          git commit -m 'Document'
+
+      - name: Push
+        uses: r-lib/actions/pr-push@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+  style:
```
Comment (dinakar29) [inline .github/workflows/pr-commands.yaml:49]: No need for style.

**Comment by dinakar29** [comment]
@waddella Can we disable CircleCI?
@gmbecker Is the `dm` package published on CRAN or is it a BioC package?

**Comment by gmbecker** [comment]
dm is CRAN.

~G

File: .github/workflows/pkgdown.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Build docs
         run: |
           options(repos = c(CRAN = "https://cloud.r-project.org/"))
-          "pkgdown" %in% install.packages || install.packages("pkgdown", upgrade = "never")
+          "pkgdown" %in% install.packages() || install.packages("pkgdown", upgrade = "never")
```
Comment (dinakar29) [inline .github/workflows/pkgdown.yaml:39]: @waddellaroche @waddella `installed.packages()` 😊

**Review by waddella — APPROVED** [review]
thanks you very much @dinakar29

**Pull request merged**